pkg/rancher-desktop/utils/_demo_marketplace_items.js

```diff
@@ -1,5 +1,31 @@
 export const demoMarketplace = {
   summaries: [
+    {
+      id:                    '',
+      name:                  'Open WebUI Extension',
+      containerd_compatible: true,
+      slug:                  'ghcr.io/rancher-sandbox/rancher-desktop-rdx-open-webui',
+      type:                  'extension',
+      publisher:             { name: 'SUSE LLC' },
+      created_at:            '2024-10-03T15:43:34Z',
+      updated_at:            '2024-10-03T15:43:34.144218728Z',
+      short_description:     'Open WebUI and Ollama packaged into a docker extension for local GenAI development',
```

Comment:
All extensions are docker extensions, so that can be removed. Especially since the extension is only visible from Rancher Desktop.

```suggestion
      short_description:     'Open WebUI and Ollama packaged into an extension for local GenAI development',
```

I wonder if this should mention `tinyllama`, but maybe that should be on the web page instead.